frontend/ui/src/reducers/data/templates/index.ts

```diff
@@ -22,7 +23,8 @@ export default function templatesReducer(state = defaultState, action: Action):
     case getType(actions.listTemplatesAction):
       return {
         ...state,
-        all: action.payload,
+        all: action.payload.templates,
+        source: action.payload.source,
       };
```

Comment:
This keeps track of the source of the templates listed,  which is needed because we fetch the list of templates when the conversation's source channel changes (as the api lists templates based on the source).